### PR TITLE
Add set of instructions to filter references

### PR DESCRIPTION
**Pull request by davidomarf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+Every reference we find must go through this proccess:
+
+- **The Finding**: We find a book, website, article, magazine, etc. that we consider
+may be useful to keep generating ideas, getting context, or keeping us interested.
+When we skim the table of contents or something similar, we should take note of what
+the book could give to ViaRe, and then the book makes it to the `finding list`.
+
+- **Note-taking**: When any member of [Daque](https://github.com/orgs/daque-dev/people)
+starts to read an item inside `finding list`, he/she must start to take notes on that
```
Comment (davidomarf) [inline README.md]: Referring as a member/contributor as "he/she" seems kinda forced. What is a more general noun? Does 'it' fits?

Comment (davidomarf) [inline README.md]: `finding list` could be either a file `FINDING.md` or a branch `finding`
I think we must agree on something.

Comment (quevangel) [reply README.md]: If I remember correctly, you can use "them".

Comment (quevangel) [reply README.md]: Or rather, "they", in this context.

Comment (davidomarf) [reply README.md]: I see more feasible a file. I strongly support that approach.

Comment (quevangel) [reply README.md]: I don't understand how it would work being a branch.

Comment (davidomarf) [reply README.md]: Ok. Fixed in last commit.

Comment (davidomarf) [reply README.md]: I can only think very abstractly about it, so we better use a file. We all agree?

Comment (quevangel) [reply README.md]: Yei.

**Pull request merged**